commons-lang/src/main/java/org/apache/commons/lang3/CharUtils.java

```diff
@@ -286,7 +286,8 @@ public static int toIntValue(final Character ch, final int defaultValue) {
      */
         public static String toString(final char ch) {
```

Comment:
indentation fixes needed to match project formatting

Reply:
@mariam606 can you please fix this in 2-3 days? Otherwise, we'll have to assign this to someone else.